src/cli/cli.cc

```diff
@@ -150,7 +164,11 @@ String getSocketHome () {
 }
 
 String getAndroidHome() {
-  auto androidHome = getEnv("ANDROID_HOME");
+  static auto androidHome = getEnv("ANDROID_HOME");
```

Comment:
Using `static` here will make unit testing / mocking harder in future.

Reply:
yeah I know what you mean, but let's punt on that for now until we can rethink this file